INSTALL.md

```diff
@@ -107,7 +107,7 @@ For proper rendering of non-latin scripts, particularly those with complicated d
 ## Dependencies
 
 For development, a style design studio is needed.
-* [Kosmtik](https://github.com/kosmtik/kosmtik) - Kosmtik can be launched with `node index.js serve path/to/openstreetmap-carto/project.mml`
+* [Kosmtik](https://github.com/kosmtik/kosmtik) - Kosmtik can be launched with `node index.js serve path/to/openstreetmap-carto/project.mml` The 0.0.17 release of Kosmtik is not enough because we need up-to-date CartoCSS and Mapnik versions. To install the current master branch of Kosmtik, you can clone the Kosmtik repository and execute `npm install` within it.
```

Comment:
This is a hard blocker for me, but there's probably a release by now.

Reply:
A kosmtik release would be great. We had asked for a new release about a year ago, but so for no reaction…

Reply:
I don’t find the reference now, but the Kosmtik developer said he won’t do releases any more (actually, the last one is from 2017), because you can easily install it directly from master at Github. So I think this is a non-issue.